tests/dev/script-user-defined-pk-parameters.R

```diff
@@ -0,0 +1,30 @@
+rm(list = ls())
+library(ospsuite)
+library(ospsuite.reportingengine)
+
+# One time initialization of PK Parameters
+ospsuite::updatePKParameter("C_max", displayName = "C max")
+ospsuite::updatePKParameter("C_max_norm", displayUnit = "kg/l" )
+ospsuite::updatePKParameter("t_max", displayUnit = "h" )
+ospsuite::updatePKParameter("AUC_tEnd", displayName = "AUC tEnd")
+myTMax <- ospsuite::addUserDefinedPKParameter("My T_MAX", standardPKParameter = StandardPKParameter$t_max, displayUnit = "min")

```

Comment:
Defining a NEW PK parameters, based on T_max. This parameter will calculate t_max starting at time = 10 min